src/graphrag_toolkit/indexing/build/vector_indexing.py

```
@@ -20,42 +20,42 @@
     @staticmethod
-    def for_vector_store(vector_store_info:VectorStoreInfoType=None, index_names=EMBEDDING_INDEXES, **kwargs):
+    def for_vector_store(vector_store_info:VectorStoreInfoType=None, index_names=DEFAULT_EMBEDDING_INDEXES, **kwargs):
         if isinstance(vector_store_info, VectorStore):
             return VectorIndexing(vector_store=vector_store_info)
         else:
             return VectorIndexing(vector_store=VectorStoreFactory.for_vector_store(vector_store_info, index_names, **kwargs))
     
     @staticmethod
-    def for_opensearch(endpoint, index_names=EMBEDDING_INDEXES, **kwargs):
+    def for_opensearch(endpoint, index_names=DEFAULT_EMBEDDING_INDEXES, **kwargs):
         return VectorIndexing(vector_store=VectorStoreFactory.for_opensearch(endpoint, index_names=index_names, *kwargs))
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)